### DISCUSSION TITLE
Projects saved as favorites do not have a search filter

### DISCUSSION THREAD

**Discussion by Angelk90** [community/community#108264] [Mobile]
### Select Topic Area

Product Feedback

### Body

Projects saved as favorites do not have a search filter

Going to the Android app on My Favorites there is no search filter.
For example, search for all those who are of a specific language.
Sort them by last commit update, ascending or descending name.
For more stars or fewer stars.

A bit like he does on his personal profile `Repositories`.

**Comment by HenSquared** [comment]
Thanks for the feedback!

That's a good idea to have additional filter items for favorite repositories. 

I've added a feature request issue internally to discuss this with the team.
